kubernetes/viai-model/viai-model.yaml.tmpl

```diff
@@ -21,6 +21,11 @@ spec:
         - name: regcred
       containers:
       - image: viai-inference-module
+        resources:
+          requests:
+            nvidia.com/gpu: 1
```

Comment:
Is this (and the following limit) expected to be hard coded to `1`?

Reply:
yes, customer can customize it to other values, for example, `0` if they don't run a GPU model. It's up to the customer to decide.
I have a customization section in the document, will mention this in the document as well.